lib/core/util.js

```diff
@@ -449,6 +450,13 @@ function toUSVString (val) {
   return hasToWellFormed ? `${val}`.toWellFormed() : nodeUtil.toUSVString(val)
 }
 
+/**
+ * @param {string} val
+ */
+function isUSVString (val) {
```

Comment:
please add a TODO to move this to webidl

Reply:
```suggestion
// TODO: move this to webidl 
function isUSVString (val) {
```